tests/intensive/sam_image_model_tests.py

```
@@ -416,5 +419,267 @@ def test_predict_without_sample_works(self):
+                )
+                self.assertIsInstance(value, Detections)
+
+        self.dataset.delete_frame_field("sam3_vid_multi")
+
+
+class TestSAM3VisualApplyModelVideo(unittest.TestCase):
+    """Verify apply_model works for SAM3 visual-mode image model on video
+    datasets (frame-level inference with box prompts)."""
+
+    @classmethod
+    def setUpClass(cls):
```

Ruff (0.15.11)

[warning] Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)